### PR TITLE
Search Page

### LINKED ISSUE #35: Search Page
image

### PR DESCRIPTION
**Pull request by ayse8888**
Hi All
I built the Search Page. #35 
For some reason, I observed some styles are broken as attached in the cards. Also in home page and about us page modals.
I think there are some duplicate class / id names and they override each other.
I believe we can fix this when everything is merged
Please correct me if I m wrong @halitbatur 

image
image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -14,7 +15,8 @@ function App() {
       <Route path="/" component={NavBar} />
       <Route exact path="/" component={Homepage} />
       <Route path="/resources" component={ResourcesPage} />
-      <Route exact path="/about" component={AboutUs} />
+      <Route path="/about" component={AboutUs} />
+      <Route path="/search" component={SearchPage} />
```
Comment (halitbatur) [inline src/App.js]: why does this and aboutus does not have the option exact ?

Comment (ayse8888) [reply src/App.js]: sorry Halit, i think i did not notice that.  i fixed both with exact path now

**Review by halitbatur — APPROVED** [review]

**Comment by ayse8888** [comment]
Hi,
I needed to make some updates on some components so I used this branch.
1. in the search component, I added email.js to send the data from adoption form to my email address
image


2. i changed some css in the search component and in the latest guests because everyone used duplicate/same class/id names and I changed mine so the css is not broken any more.
image

3. in the about us page, i added mustafa's picture

File: src/components/Search/Search.css [hunk 2]
```diff
@@ -6,8 +6,8 @@
   color: #ff3e3e;
 }
 
-.adoptButton,
-.adoptButton:hover {
+#adoptButton,
```
Comment (halitbatur) [inline src/components/Search/Search.css:9]: why are we using ids here could we use className

**Review by halitbatur — CHANGES_REQUESTED** [review]

File: src/components/Search/Search.css (hunk 2 above)
Comment (ayse8888) [reply src/components/Search/Search.css:9]: hi halit, yeah actually i had to. bcoz of the some duplicate classnames from other components, they override each other and break the css. it was like in the first picture. all css were broken. and i changed the class and ids to fix it. this one was also a classname before but i changed it to id for that reason

**Review by halitbatur — APPROVED** [review]

**Pull request merged**